src/Utility/Collections/CleanArrayPool.cs

```diff
@@ -0,0 +1,19 @@
+using System.Buffers;
+
+namespace ClassicUO.Utility.Collections
+{
+    public class CleanArrayPool<T>
```

Comment:
If desired, we could implement `ArrayPool<T>` abstract class and ignore the `clear` parameter in `Return`. Let me know if you prefer that.